docs/docs/developers/contributing.md

```diff
@@ -17,40 +17,34 @@ Active committers and contributors are invited to introduce yourself and request
 
 ## Building and Testing
 
-Chart.js uses <a href="https://gulpjs.com/" target="_blank">gulp</a> to build the library into a single JavaScript file.
-
 Firstly, we need to ensure development dependencies are installed. With node and npm installed, after cloning the Chart.js repo to a local directory, and navigating to that directory in the command line, we can run the following:
 
 ```bash
 > npm install
-> npm install -g gulp-cli
 ```
 
-This will install the local development dependencies for Chart.js, along with a CLI for the JavaScript task runner <a href="https://gulpjs.com/" target="_blank">gulp</a>.
+This will install the local development dependencies for Chart.js.
 
 The following commands are now available from the repository root:
 
 ```bash
-> gulp build                // build dist files in ./dist
-> gulp build --watch        // build and watch for changes
-> gulp unittest             // run tests from ./test/specs
-> gulp unittest --watch     // run tests and watch for source changes
-> gulp unittest --coverage  // run tests and generate coverage reports in ./coverage
-> gulp lint                 // perform code linting (ESLint)
-> gulp test                 // perform code linting and run unit tests
-> gulp test --browsers ...  // test with specified browsers (comma-separated)
+> npm run build             // build dist files in ./dist
+> npm run autobuild         // build and watch for changes
+> npm run dev               // run tests and watch for source changes
```

Comment:
I think this is already the case, but just for clarity, this also watches for test changes?

Reply:
I changed the descriptions slightly